src/test/groovy/org/prebid/server/functional/model/bidder/BidderName.groovy

```diff
@@ -4,10 +4,10 @@ import com.fasterxml.jackson.annotation.JsonValue
 
 enum BidderName {
 
-    GENERIC, RUBICON, APPNEXUS, BOGUS
+    GENERIC_ALIAS, GENERIC, RUBICON, APPNEXUS, BOGUS
```

Comment:
You can just call your bidder `ALIAS` to avoid that ugly `getValue()` conditional.